bundles/org.openhab.binding.chromecast/src/main/java/org/openhab/binding/chromecast/internal/ChromecastStatusUpdater.java

```diff
@@ -157,30 +157,32 @@ public void updateMediaStatus(final @Nullable MediaStatus mediaStatus) {
             return;
         }
 
-        switch (mediaStatus.playerState) {
-            case IDLE:
-                break;
-            case PAUSED:
-                callback.updateState(CHANNEL_CONTROL, PlayPauseType.PAUSE);
-                callback.updateState(CHANNEL_STOP, OnOffType.OFF);
-                break;
-            case BUFFERING:
-            case LOADING:
-            case PLAYING:
-                callback.updateState(CHANNEL_CONTROL, PlayPauseType.PLAY);
-                callback.updateState(CHANNEL_STOP, OnOffType.OFF);
-                break;
-            default:
-                logger.debug("Unknown media status: {}", mediaStatus.playerState);
-                break;
+        if (mediaStatus.playerState != null) {
```

Comment:
Do we want to log if it's null, similar to line 175 which logs only if new entries are added to:
https://github.com/vitalidze/chromecast-java-api-v2/blob/c1358030b1b6afa11dc2a83a0049783e7b5bef7e/src/main/java/su/litvak/chromecast/api/v2/MediaStatus.java#L41

Of course only if relevant, I don't know if this is a normal situation, or caused by some error/weird state?

Reply:
I really don't know if it's a normal situation or not. The Google docs define `playerState` as non-null, so perhaps it should be considered a bug in the NVIDIA implementation?

https://developers.google.com/cast/docs/reference/web_receiver/cast.framework.messages.MediaStatus#playerState

I was hesitant to log it, as I already see a bunch of these constantly in my log file, and I didn't want to add to the noise. I haven't yet tracked down which of my chromecast devices are causing that message. But I don't think it's the NVIDIA device.
```
2023-01-10 08:54:58.285 [DEBUG] [inding.chromecast.internal.ChromecastStatusUpdater] - MEDIA_STATUS null
```

Reply:
Let's treat it as a normal happening situation then, and refrain from logging it.